dbfit-java/build.gradle

```diff
@@ -62,7 +72,7 @@ subprojects {
         testCompile "org.hamcrest:java-hamcrest:2.0.0.0"
         testCompile "commons-io:commons-io:2.4"
         testRuntime 'com.dbdeploy:dbdeploy-ant:3.0M3'
-        integrationTestCompile "dummy:fitnesse-standalone:${fitNesseVersion}@jar"
+        integrationTestCompile "org.fitnesse:fitnesse:${fitNesseVersion}:standalone@jar"
```

Comment:
Sometimes (almost always) we're using an intermediate fitnesse build (not the official one published to maven). Would it be still possible to do that with the updated notation?